lib/clevertap_plugin.dart

```diff
@@ -755,71 +773,110 @@ class CleverTapPlugin {
   }
 
   ///Feature Flags
+  @Deprecated(
+      "This method is deprecated since v1.3.0. Use getCleverTapID() instead")
+
   ///Returns boolean value of Feature Flag
   static Future<bool?> getFeatureFlag(String key, bool defaultValue) async {
     return await _dartToNativeMethodChannel.invokeMethod(
         'getFeatureFlag', {'key': key, 'defaultValue': defaultValue});
   }
 
   ///Product Config
+    @Deprecated(
+      "This method is deprecated since version 1.5.7 and will be removed in the future versions of this SDK.")
```

Comment:
@akashvercetti We should set the next Flutter SDK version as 1.6.0 so change the comments here